src/app/screens/SignUp/index.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import i18next from 'i18next';
+import { useForm } from 'react-hook-form';
+
+import styles from './styles.module.scss';
+import WoloxImg from './assets/wolox-logo.png';
+
+interface User {
+  first_name: string;
+  last_name: string;
+  email: string;
+  password: string;
+  password_confirmation?: string;
+  locale?: string;
+}
+
+const SignUp: React.FC = () => {
```

Comment:
Follow tech guide, we declare componentss with `function Signup() { ...`
Also, remove the type `React.FC`, if you declare a component the type is inferred